packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -290,3 +292,5 @@ module.exports = {
     hints: false,
   },
 };
+
+module.exports = applyPlugins(base, ['typescript'], { path, paths });
```

Comment:
Maybe expand the whitelisted plugins into multiple lines starting from the first plugin, to minimize future diffs?
```
module.exports = applyPlugins(base, [
  'typescript',
], { path, paths });
```
Diff:
```diff
 module.exports = applyPlugins(base, [
   'typescript',
+  'sass',
 ], { path, paths });
```
Versus:
```diff
-module.exports = applyPlugins(base, ['typescript'], { path, paths });
+module.exports = applyPlugins(base, [
+  'typescript',
+  'sass',
+], { path, paths });
```